addons/knowledge/models/res_users.py

```diff
@@ -39,7 +39,7 @@ def _generate_tutorial_articles(self):
                 'icon': "👋",
                 'internal_permission': 'none',
                 'favorite_ids': [(0, 0, {
-                    'sequence': 0,
+                    'sequence': 1,
```

Comment:
Why this change ?

Reply:
Since I added sample data in the favourite articles, this tutorial article was not the one opened by default anymore.
I felt like this welcome article should be the one opened by default, even if there are sample favourite articles, so I made this change.
However, since you suggested to move these samples to a demo file, I can revert this modification if it does not matter that the tutorial article is not the one opened by default when there is demo data.